### ISSUE TITLE
Surface catalogue 0.3 isn't loading.

### ISSUE THREAD

**Issue by jeremylightsmith** [surface-ui/surface_catalogue#31]
I've just started a new phoenix project, added surface, and it all works if I use:

```elixir
      {:phoenix_live_view, "~> 0.16.0"},
      {:surface, "~> 0.6.0"},
      {:surface_catalogue, "~> 0.2.0"}
```

I see this when I open up the catalogue:

<img width="1134" alt="image" src="https://user-images.githubusercontent.com/15301/156270137-d3febea5-b57b-4293-b3a9-2e020dfe3e39.png">

But the catalog won't show any components if I use:

```elixir
      {:phoenix_live_view, "~> 0.17.6"},
      {:surface, "~> 0.7.0"},
      {:surface_catalogue, "~> 0.3.0"}
```

I think there is an error connecting the socket, maybe?

<img width="1203" alt="image" src="https://user-images.githubusercontent.com/15301/156269986-5bddb4a8-4ffb-4527-9243-f0c2c893c421.png">

I'm trying to use the catalog on my project, which happens to be on live view 0.17.6, so it's not letting me use this awesome tool.

**Comment by jeremylightsmith** [comment]
Here's the my code. It's just a greenfield project with a couple example components in it.

[play.zip](https://github.com/surface-ui/surface_catalogue/files/8165561/play.zip)

**Comment by crova** [comment]
I'm having a similar issue on a project I'm working on:
![image](https://user-images.githubusercontent.com/13876051/156365862-b19a44b5-acf2-45e8-ade2-5eae22bd0bd9.png)

Screenshot from Firefox dev tool, looks like the OP was on Chrome (I get the same error message as him on Chrome).

Let me know if I can provide/help with anything else here.

**Comment by noozo** [comment]
Can confirm. I reported it to msaraiva a few weeks ago on the slack channel. I think i managed to track it as far as `https://github.com/surface-ui/surface_catalogue/blob/master/lib/surface/catalogue/util.ex` in `components_reducer` not being able to read the components, but didn't pursue any further.

**Comment by clsource** [comment]
In my investigations the `component_tree.ex` file receives the params and renders the list of components.
This is failing due to `templates` not beign loaded and crashing the js execution.

![imagen](https://user-images.githubusercontent.com/292738/157365035-e6396d81-ec5f-441f-a178-c061df5c526b.png)

![imagen](https://user-images.githubusercontent.com/292738/157365062-8be19c4b-e530-4767-9a50-62fcd7526d66.png)

**Comment by msaraiva** [comment]
Fixed in `v0.4.0`
